connpolicy_test.go

```diff
@@ -0,0 +1,177 @@
+	serverCertBase64 = "MIIF2DCCA8CgAwIBAgICEAEwDQYJKoZIhvcNAQELBQAwdTELMAkGA1UEBhMCR0IxFzAVBgNVBAgMDlVuaXRlZCBLaW5nZG9tMQ8wDQYDVQQHDAZMb25kb24xEjAQBgNVBAoMCUFDTUUgSW5jLjEZMBcGA1UECwwQQW52aWwgRGVwYXJ0bWVudDENMAsGA1UEAwwEcm9vdDAeFw0xOTA4MzAxMDQ1MzNaFw0yOTA3MDgxMDQ1MzNaMGkxCzAJBgNVBAYTAkdCMRcwFQYDVQQIDA5Vbml0ZWQgS2luZ2RvbTESMBAGA1UECgwJQUNNRSBJbmMuMRkwFwYDVQQLDBBBbnZpbCBEZXBhcnRtZW50MRIwEAYDVQQDDAlsb2NhbGhvc3QwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQChI+Q55qCDCOWxSFwcaMyVnY4G1Kjgo4WIiYVIJ1mex1q5XNjPGc+hD942KyareQZOXA/Xe0wbna9mZosnmqvuSA78EIZ3TpLZdcbc6oZx47yY9Sb/0Pc1KL+Wx12KqivU86ZEduQTW+tYULma64OOKk5qmp3FaZsoNwUBI6cOiNBrvxqlMjDsMBAhMBgrRGhcQwAmK++irnB2GCtG6l58osvrdv6WTYZ+EpEKSHm3WjiMGBG0DKi4tTAdNUQET2siCM7aTlE03yDqVT5f/cBit6R7SPHSR+xeZF0emk9fegwEBYjJD7r5ItVXZIm1N0a1VCnZxky29fV9Cn01cKEnAgMBAAGjggF8MIIBeDAJBgNVHRMEAjAAMBEGCWCGSAGG+EIBAQQEAwIGQDArBglghkgBhvhCAQ0EHhYcR2VuZXJhdGVkIFNlcnZlciBDZXJ0aWZpY2F0ZTAdBgNVHQ4EFgQUnQSqjMYpErYOmO5U4fxbBl8oAkUwgbIGA1UdIwSBqjCBp4AUHTg2XTOhyEBEkwSs9A6bkevOfoOheaR3MHUxCzAJBgNVBAYTAkdCMRcwFQYDVQQIDA5Vbml0ZWQgS2luZ2RvbTEPMA0GA1UEBwwGTG9uZG9uMRIwEAYDVQQKDAlBQ01FIEluYy4xGTAXBgNVBAsMEEFudmlsIERlcGFydG1lbnQxDTALBgNVBAMMBHJvb3SCFGOtyOOeL8byY03yYIqyKW1SQMKXMA4GA1UdDwEB/wQEAwIFoDATBgNVHSUEDDAKBggrBgEFBQcDATAyBgNVHR8EKzApMCegJaAjhiFodHRwczovLzEyNy4wLjAuMTo4MjAwL3YxL3BraS9jcmwwDQYJKoZIhvcNAQELBQADggIBACXuuAVzNuqdVt1irPHnWcDOkwJoFAIeVmI0/BZPaMPEZJc/vd3/66+sRJ5pX7j8Bl6CCMT6RZauQk7engiAfgbHmRt0Cngo+0K4KChcbsG5RWsjB7fXtI/Xup8D0pfNwDUqPgQt50P0Gz1MOgFPEDtb0OcIiKIEVGEa+noHMQkspAKnINzgROApjljGcwgpSenU7Y9r4NkcZKsZkHrCp5HL3cZCJAI9LwUAE40gNAo10lfQrzgVzeCj1mpfghrYVGU9AO4wcyad7zBHbb0ZhudP39RsdfjotngiB1ANsa0lfuKXZ2cXBZFsv3JZOfRQz02UyPqg3X1yaxktYuaxc/KejeceTu73wkM/ir3topUg616KC7YIqtUq2Qu3n0vhBSwY+fzEudbfxp2nuqefqmnYuENwkNxdiXuqqWQ5k8ISBgECVc8VY27441wU7lTJFz/CMpU7Z9jx5Wf3YlMALHRVyyzdWJGBW1/wXxaGvBRzaEDAl4VqsRX1mij/4xAcToTQhCzSzKE/MGokMLpB+HrWCyqG+55wcwVBcyA7oFYs3f7a7BynPuOkV64rNO7ouaxbXyhpsMmE0kjvEIJD7XGyLuOJa6lThlEokLK/StXv620PPQ5zrGGHG6R4IQquToXxcdz4h5pdpCaQ7LTGG6BBm4bRNlcWobIX1DzLX+vn"
```

Comment:
Prefix these with `test` so that we can make sure it's clear they're only used in tests (even though it's in a test file, it's just good form I feel like) -- for example, `testRootCA`.